### ISSUE TITLE
Proposed style guide additions (ongoing list)

### ISSUE THREAD

**Issue by mst3k4ever** [AgoraDoc/agora-style-guide#18]
This is just a preliminary note that I am going to try to add a section of using conjunctive adverbs (adverbs that link to independent sentences or clause, such as "therefore"); this came up in a doc, and a writer comment made me rethink my edit.

**Comment by mst3k4ever** [comment]
I promise to spell "conjunctive" correctly when I do the actual update.  :(

**Comment by mst3k4ever** [comment]
Okay, I have changed my mind. I am going to use this issue to collect my proposed additions. Once I have a bunch, I will write them up and propose the changes. I will just continue to edit this comment for now. Cheers.

1. conjunctive adverbs
2. comma splices

**Comment by DITAWriter** [comment]
I'll just add that I think these are good topics to tackle. I have tried to be consistent with my edits in this area, but I think Bill is better qualified that I am to set the writing standard for this going forward.
I'll just add that if you (Bill) can find and add references to appropriate sections from other companies Style Guides (like Microsoft, for example), that would lend further credence to any argument you make.
Cheers!
    On Monday, September 28, 2020, 10:51:46 a.m. EDT, mst3k4ever <notifications@github.com> wrote:  
 
 


Okay, I have changed my mind. I am going to use this issue to collect my proposed additions. Once I have a bunch, I will write them up and propose the changes. I will just continue to edit this comment for now. Cheers.
   
   - conjunctive adverbs
   - comma splices
